whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -117,16 +125,12 @@ public String call() {
         });
     }
 
-    private void validateJdbcUrl(final String user, final String password) {
-        if (!user.equals(password) && !user.equals(reverse(password))) {
-            throw new IllegalArgumentException("dummifier runs on non-production environments only (user==password)");
+    private void validateEnvironment(final String env) {
+        if (!AVAILABLE_ENVIRONMENTS.contains(env)) {
+            throw new IllegalArgumentException("dummifier runs on non-production environments only");
```

Comment:
if it will throw error only in production , can you flip the condition, if it is production throw error?

Reply:
If I check that the user is just adding PROD is less secure that if I check that the user is specifying exactly the correct env.
For example: env=pro -> This will give True if we are just checking env.equals(prod)

Reply:
Can you introduce an enum for environment? And use enum equals? You need to first convert the user input to enum.

Reply:
Ok, good idea, this should work yes.

Reply:
Done